include/uapi/sound/sof/tokens.h

```diff
@@ -67,6 +67,7 @@
 /* Token retired with ABI 3.2, do not use for new capabilities
  * #define SOF_TKN_COMP_PRELOAD_COUNT		403
  */
+#define SOF_TKN_COMP_REF			404
```

Comment:
what is this?

Reply:
@plbossart as explained above, this is the connection point for this client into the host "physical" topology.